web/data/permissions.go

```diff
@@ -26,7 +26,6 @@ var blackList = map[string]bool{
 	consts.Files:            readable,
 	consts.Jobs:             readable,
 	consts.Notifications:    readable,
-	consts.Queues:           readable,
```

Comment:
Why this change?

Reply:
I removed completely the `io.cozy.queues` doctype. It wasn't really required looking back.